sdks/python/apache_beam/testing/util.py

```diff
@@ -179,11 +179,12 @@ def _equal(actual):
           expected_list.remove(element)
         except ValueError:
           raise BeamAssertException(
-              'Failed assert: %r == %r, expected missing element %r' %
+              'Failed assert: %r == %r, right side missing at least element %r' %
```

Comment:
This line exceeds 80 characters. Please fix. Thanks to this PR I realized our lint checker stopped catching this error: https://issues.apache.org/jira/browse/BEAM-9058. Fix is in flight (https://github.com/apache/beam/pull/10510).